tests/integration/docusaurus/tutorials/quickstart/v1_pandas_quickstart.py

```diff
@@ -0,0 +1,56 @@
+"""
+This guide demonstrates how to use Great Expectations in a quick-start
+notebook environment with Pandas. It is useful to explore and understand how Great Expectations
+works, using your own data.
+
+Note:
+- Do not follow this workflow when you are building a durable workflow because it uses
+ephemeral assets.
+- Do not use this workflow for embedding in a pipeline or CI system, because it uses an
+iterative process for trying and refining expectations.
+"""
+
+
+# <snippet name="tests/integration/docusaurus/tutorials/quickstart/v1_pandas_quickstart.py import_gx">
+import great_expectations as gx
+import great_expectations.expectations as gxe
+
+# </snippet>
+
+# Set up
+# NOTE: Context is a singleton now. Once the context has been set instantiated in a session
+# <snippet name="tests/integration/docusaurus/tutorials/quickstart/v1_pandas_quickstart.py get_context">
+context = gx.get_context()
+# </snippet>
+
+# <snippet name="tests/integration/docusaurus/tutorials/quickstart/v1_pandas_quickstart.py connect_to_data">
+batch = context.sources.pandas_default.read_csv(
```

Comment:
I'm curious if there is a local csv in this repo that we can use instead of this URL.

Reply:
there probably is but I think we want the URL since this is quickstart code and should be run without any prework

Reply:
Agree--this is designed for someone who as run pip install, but not cloned the repo.

Reply:
I see, that makes sense. 

If we want newer data we could upload that to the gx_tutorial repo. We could also put it in this repo if we want and access like so:

https://raw.githubusercontent.com/great-expectations/great_expectations/develop/tests/test_sets/taxi_yellow_tripdata_samples/sqlite/yellow_tripdata.db

Reply:
The new data is all in parquet - will bring up during standup